docs/reference/setup-tools/kubeadm/kubeadm-alpha.md

```diff
@@ -118,7 +119,7 @@ You can create all required kubeconfig files with the `all` sub command, or sele
 
 ## kubeadm alpha phase controlplane {#cmd-phase-controlplane}
 
-You can create all required static pod files for control plane components with the `all` sub command, 
+You can create all required static Pod files for the control plane components with the `all` subcommand, 
```

Comment:
I don't see why you are capitalizing Pod all over unless it is that way in the rest of the docs.?.?

/cc @Bradamant3

Reply:
@timothysc It's the correct convention, yes.